### PR TITLE
rtpengine: Fix set selection

### PR DESCRIPTION
**Pull request by smititelu**
When set_rtpengine_set() config function is called.

#### Pre-Submission Checklist

- [X] Commit message has the format required by CONTRIBUTING guide
- [X] Commits are split per component (core, individual modules, libs, utils, ...)
- [X] Each component has a single commit (if not, squash them into one commit)
- [X] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:

- [X] PR should be backported to stable branches
- [X] Tested changes locally
- [ ] Related to issue #XXXX (replace XXXX with an open issue number)

#### Description

After calling set_rtpengine_set(), the new set is not set for further rtpengine delete(). This is because active_rtpp_set is set to default_rtpp_set when msg id != current_msg_id (which is set once by set_rtpengine_set())

I've removed the msg id != current_msg_id condition, and initialized active_rtpp_set in mod_init().

Basically tested locally with 2 sets with 1 rtpengine each.

### CODE REVIEW

**Comment by miconda** [comment]
At the first sight, does not seem good.

That protection with msg_id is to ensure that a rtpengine group (set) is used only for the SIP message it was set for.

Now it seems you remove that protection and setting a rtpengine group stays until is set again.

Anyhow, the end of the call (BYE) can be handled by a different SIP worker process than the one which handled the corresponding INVITE, so you can't really have (expect) the rtpengine group to be set to same value from INVITE till BYE. You should store it inside a dlg variable (if you use dialog module) or in a hashtable. It is also an option to add it as a param to record-route header.

**Comment by smititelu** [comment]
Thank you for the feedback! Closing this PR.
